src/App.tsx

```diff
@@ -1,31 +1,36 @@
 import React from 'react'
 import SearchPage from './components/SearchPage'
 import MyCollectionsPage from './components/MyCollectionsPage'
+import SelectLanguage from './components/SelectLanguage'
 import { Routes, Route, Link } from 'react-router-dom'
 import { CollectionProvider } from 'hooks'
 import { QueryClient, QueryClientProvider } from 'react-query'
 import { ReactQueryDevtools } from 'react-query/devtools'
 import { ImSearch } from 'react-icons/im'
 import { HiCollection } from 'react-icons/hi'
 import tw from 'twin.macro'
+import './i18n/i18n'
+import { useTranslation } from 'react-i18next'
 
 const queryClient = new QueryClient()
 
 const App = () => {
+  const { t } = useTranslation()
   return (
     <div>
       <Nav>
         <UnorderedList>
           <Link to="/">
             <NavLink>
-              My Collection <CollectionIcon />
+              {t('pages.myCollection')} <CollectionIcon />
```

Comment:
for translations, the key should just be the default translation:

```suggestion
              {t('Collection Page')} <CollectionIcon />
```